parser/parser.go

```diff
@@ -180,6 +181,30 @@ func Normalize(input string) (result string, err error) {
 	return
 }
 
+func Split(input string) (result []string, err error) {
```

Comment:
It might be best to call this `SplitWithParser` (matching the naming in the C library), since there are use cases where its preferred to use `SplitWithScanner`, see https://github.com/pganalyze/libpg_query/blob/15-latest/pg_query.h#L85